spec/rails_helper.rb

```diff
@@ -75,6 +75,13 @@
   end
 
   config.include ActionMailerMatchers
+  
+  
+  # needed for some CVE tests
+  config.include Minitest::Assertions
+  config.include ActiveSupport::Testing::Assertions
```

Comment:
It might be faster to include them in the individual specs themselves (if they truly are needed one-off).

Reply:
I vaguely remember trying that and it not working. I could try again.

Reply:
I moved these into the associated CVE spec.